api/rpc_test.go

```diff
@@ -177,21 +177,35 @@ func TestAuthedRPC(t *testing.T) {
 				stats, err := rpcClient.DAS.SamplingStats(ctx)
 				require.NoError(t, err)
 				require.Equal(t, expected, stats)
+
+				expectedResp := &state.TxResponse{}
+				server.State.EXPECT().SubmitTx(gomock.Any(),
+					gomock.Any()).Return(expectedResp, nil)
+				txResp, err := rpcClient.State.SubmitTx(ctx, []byte{})
+				require.NoError(t, err)
+				require.Equal(t, expectedResp, txResp)
```

Comment:
Not necessary as we have an example of a read-level permissions endpoint :) But thank you

Reply:
Resolved